hw/ip/otbn/dv/otbnsim/test/simple/li/li.s

```diff
@@ -0,0 +1,35 @@
+/* Copyright lowRISC contributors. */
+/* Licensed under the Apache License, Version 2.0, see LICENSE for details. */
+/* SPDX-License-Identifier: Apache-2.0 */
+/*
+    Checks for li support
+
+    This is really a test of the assembler, which is supposed to expand the li
+	pseudo-instruction into addi/lui as necessary. Of course, checking that the
+	expansion was correct is a little difficult, unless you can run the
+	generated instructions... which is what we can do for ISS testing. So the
+	checks seem to fit quite well here.
```

Comment:
ALARM! TABS! :smile: ([We don't like precision.](https://www.youtube.com/watch?v=SsoOG6ZeyUI))

Reply:
Oh no! I think I need to go and bash my `asm-mode` config...